src/s3/commands/downloadFileAs.ts

```diff
@@ -119,9 +120,17 @@ export async function downloadFileAsCommand(
     const sourcePath = readablePath(node)
 
     await telemetry.s3_downloadObject.run(async () => {
-        const saveLocation = await promptForSaveLocation(file.name, window)
+        let lastUsedDownloadPath: string | undefined
+        const storedPath = globals.context.globalState.get('workbench.explorer.downloadPath')
+        if (typeof storedPath === 'object' && isNonNullable(storedPath) && hasStringProps(storedPath, 'fsPath')) {
+            lastUsedDownloadPath = path.dirname(storedPath.fsPath)
+        }
+
+        const saveLocation = await promptForSaveLocation(file.name, window, lastUsedDownloadPath)
         if (!saveLocation) {
             throw new CancellationError('user')
+        } else {
+            globals.context.globalState.update('workbench.explorer.downloadPath', saveLocation)
```

Comment:
need at least a comment here explaining the significance of this. This is not a casual line of code.

Reply:
This is saved in the Toolkit's globalStorage and has been renamed to `aws.latestDowloadPath`